spec/rails_helper.rb

```diff
@@ -4,7 +4,12 @@
 require_relative "./support/test_helpers"
 require_relative "./support/adobe_campaign_stub_helpers"
 
-SimpleCov.start
+SimpleCov.start 'rails' do
```

Comment:
it looks like the single quotes here breaks standardrb

Reply:
sigh... yes. fixed.